src/app/containers/Disclaimer/index.stories.jsx

```diff
@@ -33,8 +33,12 @@ const DISCLAIMER_FIXTURE = {
       text: 'Android',
       url: 'https://play.google.com/store/apps/details?id=uk.co.bbc.russian',
     },
-    para5:
-      '. Грузите его на ваш девайс и продолжайте получать новости от Би-би-си.',
+    para5: '. Вы можете также подписаться на наш канал в ',
+    para6: {
+      text: ' Telegram',
```

Comment:
oh! I have spotted something!! You have 2 spaces between para5 and para6. I suggest you remove the one from the para6 to keep it consistent with other links: 
> 'Telegram'